pkgs/applications/audio/rofi-mpd/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, python37Packages, fetchFromGitHub }:
```

Comment:
`python37Packages` is very specific. `python37Packages` should be fine here?

Reply:
Sorry I'm not quite sure what you mean? Just python3Packages in the second instance?

Reply:
Sorry, copy-and-paste mistake. Yes, I meant to use `python3Packages`.

Reply:
if you really want flexibility, you would just pass pythonPackages, and in all-packages, you would pass which version you would want. `pythonPackages = python37Packages`

Reply:
Since this package only works with Python >= 3.6 I won't do this, but I'll note this for the future. Thanks.